File: frontend/app/src/main/java/com/example/easypark/easyparkfinal/utils/MessageBuilder.java
package com.example.easypark.easyparkfinal.utils;

import android.content.Context;
import android.widget.Toast;

public class MessageBuilder {

    public static void exibirMensagem(String mensagem, Context context){
        Toast.makeText(context,mensagem,Toast.LENGTH_SHORT).show();
    }
}

File: MobileTruck/app/src/main/java/easypark/com/mobiletruck/utils/MessageBuilder.kt
package easypark.com.mobiletruck.utils

import android.content.Context
import android.widget.Toast


class MessageBuilder {

    companion object{
        fun exibirMensagem(mensagem: String, context: Context) {
            Toast.makeText(context, mensagem, Toast.LENGTH_SHORT).show()
        }
    }
}